.utility/update-docs-repo.sh

```diff
@@ -13,8 +15,8 @@ git clone --quiet --branch=master git@github.com:loklak/dev.loklak.org.git lokla
 
 cd loklak_docs
 
-if ! git subtree pull --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree" ; then
-  git subtree add --prefix=raw/server ../loklak_server documentation --squash -m "Update server subtree"
+if ! git subtree pull --prefix=raw/server $LOC documentation --squash -m "Update server subtree" ; then
```

Comment:
Also, instead of putting `if !...` would it be better to put
```sh
if git subtree pull ... ; then
    echo "Pulled subtree ... "
else
    echo "Failed to pull subtree ... Adding subtree first"
    git subtree add ...
fi
```